db/bigtable_eth1.go

```diff
@@ -3453,10 +3460,10 @@ func (bigtable *Bigtable) SaveMethodSignatures(signatures []types.MethodSignatur
 }
 
 // get a method signature by it's hex representation
```

Comment:
**Minor:** Comment now is wrong as this supports both methods and events.

Reply:
changed